index.bs

```diff
@@ -1,317 +1,307 @@
-
-While these forms of isolation help,
-they do not guarantee that isolated content does not need to be rendered at all.
-Ideally there would be a way for the developer to specify
-that specific parts of the DOM need not be rendered,
-and pair that with a guarantee that when later rendered,
-it would not invalidate more than a small amount of style, layout, or paint
-in the rest of the document.
-
-Disclaimer {#disclaimer}
-----------
-
-As the proposed features evolve,
-several competing API shapes might be considered at the same time,
-the decisions on particular behaviors might not be finalized,
-and some documentation may be out of date.
-
-For the latest implemented behavior and API state,
-please consult the cheatsheet.
-
-For behaviors being discussed,
-as well as questions and other discussions,
-please look over the issues.
-
-The rest of this document talks about one particular implementation option.
-Whether or not this is the final proposed set of features is yet undecided.
-
-
-Controlling Rendering Of A Subtree {#controlling-rendering}
-==================================
-
-Preventing Rendering: the 'render-subtree' property {#render-subtree}
----------------------------------------------------
+* <a href=#f2>[2]</a> and <a href=#f3>[3]</a> usually means that such content
+    is not available to user-agent features, such as find-in-page functionality.
+    Also, content that is merely placed off-screen may or may not have rendering
+    cost (it depends on browser heuristics), which makes the technique
+    unreliable.
+
+* Caching intermediate rendering state is hard work, and often has performance
+    limitations and cliffs that are not obvious to developers.  Similarly,
+    relying on the browser to avoid rendering for content that is clipped out or
+    not visible is sometimes not reliable, as it's hard for the browser to
+    efficiently detect what content is not visible and does not affect visible
+    content in any way.
+
+Previously adopted web APIs, in particular the 'contain' and 'will-change' CSS
+properties, add ways to specify forms of rendering isolation or isolation
+hints, with the intention of them being a mechanism for the web developer to
+help the browser optimize rendering for the page.
+
+While these forms of isolation help, they do not guarantee that isolated
+content does not need to be rendered at all.  Ideally there would be a way for
+the developer to specify that specific parts of the DOM need not be rendered,
+and pair that with a guarantee that when later rendered, it would not
+invalidate more than a small amount of style, layout, or paint in the rest of
+the document.
+
+Definitions {#definitions}
+===========
+
+* <dfn export>off-screen</dfn>: an element is considered to be off-screen if
+    its border box does not intersect the viewport with a fixed, user-agent
+    specified, margin. 
+    <div class=note>
+      The the user-agent is free to consider a fixed margin
```

Comment:
Maybe the UA should be allows to increase the margin to account for scroll velocity?

Reply:
Yeah I was thinking about this. I think it doesn't really make sense for us to put any arbitrary limits on this, but at the same time I don't want to focus the discussion on this margin. I'll just write that it's up to the user-agent to determine the margin

Reply:
ok